src/styles/index.css

```diff
@@ -0,0 +1,198 @@
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.nav__img {
+  display: block;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  font-family: Roboto, Arial, sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+}
+
+.nav__list {
+  display: flex;
+  list-style: none;
+  margin: 0;
+  padding: 0;
+}
+
+.nav__item {
+  margin-right: 20px;
+}
+
+.nav__item:last-child {
+  margin-right: 0;
+}
+
+.nav__link {
+  display: flex;
+  align-items: center;
+  height: 60px;
+  color: #000;
```

Comment:
The color for the `.nav__link` should be `#060b35` instead of `#000` to match the task requirements【28:0†index.css】.